src/gwt/src/org/rstudio/studio/client/workbench/views/source/Source.java

```diff
@@ -3132,6 +3141,7 @@ public void onSelection(SelectionEvent<Integer> event)
       if (event.getSelectedItem() >= 0)
       {
          activeEditor_ = editors_.get(event.getSelectedItem());
+         consoleEditorHadFocusLast_ = false;
```

Comment:
Okay, I'll hold merging this until after we branch then.

On Fri, Jan 22, 2016 at 10:38 PM, Kevin Ushey <notifications@github.com>
wrote:

> In
> src/gwt/src/org/rstudio/studio/client/workbench/views/source/Source.java
> <https://github.com/rstudio/rstudio/pull/558#discussion_r50615151>:
>
> > @@ -3132,6 +3141,7 @@ public void onSelection(SelectionEvent<Integer> event)
> >        if (event.getSelectedItem() >= 0)
> >        {
> >           activeEditor_ = editors_.get(event.getSelectedItem());
> > +         consoleEditorHadFocusLast_ = false;
>
> I think it's fine to leave it. The issue @wch <https://github.com/wch>
> pointed out occurs only when you attempt to use an interactive addin when
> there is no source document available (ie, only the console is open, and
> the source window is hidden)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rstudio/rstudio/pull/558/files#r50615151>.
>